src/plays/github-user-finder/components/InputCard.css

```diff
@@ -0,0 +1,182 @@
+#input {
+  height: 4rem;
+  width: 20rem;
+  display: flex;
+  padding: 1rem;
+  margin: 1rem;
+  outline: none;
+  border-radius: 20px;
+  background: #ffffff;
+  box-shadow: 20px 20px 60px #d9d9d9, -20px -20px 60px #ffffff;
+}
+
```

Comment:
Some of these CSS break will be fixed when you take care of the overriding comment I gave

![image](https://user-images.githubusercontent.com/3633137/216959709-25af3991-64f0-4595-ac38-f2f40b0e946c.png)

Reply:
@ANKIT9761 I think I was not clear about giving the comment. What I meant is add an additional style to this like:

```html
<div className="play-details-body guf">
```

Now you can use the `.guf` as a prefix for all the styles in all CSS files, like

```css
.guf button{}

.guf #clip{}

// like these
```

Got it? Please make the changes.

Reply:
Ok thanks finally made the changes just a doubt why don't we use css modules to avoid these name collision or css modules doesn't help here curious about it.